x/harvest/keeper/borrow.go

```diff
@@ -17,6 +17,13 @@ func (k Keeper) Borrow(ctx sdk.Context, borrower sdk.AccAddress, coins sdk.Coins
 		return err
 	}
 
+	for _, coin := range coins {
```

Comment:
I agree here, but worth noting that this contradicts the document which states: "**Note**: The AccrueInterest function is not called in the begin blocker, but rather whenever a deposit/withdraw or borrow/repay message is processed, or when the interest rate model is updated"

Reply:
See https://github.com/Kava-Labs/kava/pull/720#discussion_r525052908. Instead of using a list of borrowed coins latest version uses the `InterestRateModels` in the params/store, respectively, to determine which assets to accrue interest on. This resolves a potential edge case where:
1. User borrows asset A
2. User accrues interest according to asset A's interest rate model
3. Asset A's interest rate model is removed from params and therefore removed from store
4. BeginBlocker iterates over total borrows and treats the outstanding borrow of asset A as an indication that interest should be accrued
5. AccruedInterest can't find asset A's interest rate model in the store and panics

In latest, interest rate models can be removed from params/store while borrow balances are outstanding without issues. The outstanding borrows will just stop earning interest until the asset's interest rate model is added back to params via governance.